pyEpiabm/pyEpiabm/tests/test_unit/test_routine/test_file_population_config.py

```diff
@@ -23,6 +23,8 @@ def setUp(self) -> None:
     def test_make_pop_no_file(self, mock_log):
         """Tests for when no file is specified.
         """
+        pe.Parameters.instance().household_size_distribution = []
```

Comment:
Is this needed in every test? Could it go in the setUp() instead? (And why do we need it everytime remind me?

Reply:
Have moved it to the set up. I added since the household_size_distribution it specified in the parameters file, but the way it is now set up this will call the method in InitialHouseholdSweep, so would need mocking every time. Therefore I set the parameter list to empty in all but one test to restrict the tests to using the methods within file population config.